### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN043.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#91] [General]
# https://blog.3place.cn/3place/lmyn/LMYN043.html

https://blog.3place.cn/3place/lmyn/LMYN043.html

**Comment by picklecai** [comment]
从天空直落下来，
落到湿婆的身上和头顶。 （1.42.4）

她被湿婆的乱发缠绕住，
她无法找到一条路逃走。
于是恒河女神就在那里，
回旋奔流，久历春秋。 （1.42.5）

乱发缠绕和逃走，这个设定有意思。谁想长留人间呢？恒河大概被骗“去人间看一眼就行”。结果被乱发缠绕 住了，跑不掉了。

**Comment by picklecai** [comment]
从天堂落到湿婆头上，
然后又落上了大地，
水流迸发出澎湃涛声，
就在那里流转不息。 （1.42.7）

这得多大的冲量？不可计算了。

**Comment by picklecai** [comment]
他们来来往往地走动，
分散地站在四面八方。 （1.42.9）

成群的神仙都来到，
他们具有无量辉光，
渴望看一看恒河下凡，
大地上奇迹无双。 （1.42.10）

神仙也是吃瓜群众，爱凑热闹。

**Comment by picklecai** [comment]
有的地方，水流得弯曲迅速，
有的地方，水流又被阻住，
有的地方，水流得弯曲摇荡，
有的地方，水又缓缓地流出。 （1.42.14）

有的地方，水流同水流
互相撞击，碰在一起，
一刹那间，流上天去，
但一转眼，又落下平地。 （1.42.15）

这是借机写真实的恒河。

**Comment by picklecai** [comment]
有些神仙受到诅咒，
从天宫里掉到地上，
他们在这里举行水祭，
他们的罪孽都洗光。 （1.42.18）

他们用闪光的圣水，
洗涤了自己的罪恶；
他们又飞上了天空，
各自走回自己的宫阙。 （1.42.19）

吃瓜群众也不只是为了吃瓜。恒河圣水的传说，大概就源于这个故事吧。

**Reply by picklecai** [reply]
光辉的恒河就跟到哪里，
这一条一切江河的魁首，
能够把一切罪孽都洗涤。” （1.42.24）

圣水的现实基础是：恒河是一切江河的魁首。
古人：这条河怎么这么大？这么壮阔？这么长？从没见过这样的河。那它一定很神奇，大概什么都能洗干净吧。

**Comment by picklecai** [comment]
“紧那罗”在梵语中为“人非人”之意。他形状和人一样，但头上生一只角，所以称为“人非人”，善于歌舞，是帝释的乐神。

**Comment by picklecai** [comment]
“乾达婆”是一种不吃酒肉、只寻香气作为滋养的神，是服侍帝释的乐神之一，身上发出浓冽的香气。“乾达婆”在梵语中又是“变幻莫测”的意思，魔术师也叫“乾达婆”，海市蜃楼叫做“乾达婆城”。香气和音乐都是缥缈隐约，难以捉摸。

**Comment by picklecai** [comment]
“夜叉”是佛经中的一种鬼神，有“夜叉八大将”、“十六大夜叉将”等名词。“夜叉”的本义是能吃鬼的神，又有敏捷、勇健、轻灵、秘密等意思。《维摩经》注：“什曰：‘夜叉有三种：一、在地，二、在空虚，三、天夜叉也。’”现在我们说到“夜叉”都是指恶鬼。但在佛经中，有很多夜叉是好的，夜叉八大将的任务是“维护众生界”。

**Reply by picklecai** [reply]
夜叉果然不代表恶。上次那个被神仙捉弄了的夜叉，儿子没死之前，就美丽、活力。

**Comment by picklecai** [comment]
大蛇，大概是龙。

“龙”是指龙神。佛经中的龙，和我国传说中的龙大致差不多，不过没有脚，有时大蟒蛇也称为龙。事实上，中国人对龙和龙王的观念，主要是从佛经中来的。佛经中有五龙王、七龙王、八龙王等等名称。古印度人对龙很是尊敬，认为水中生物以龙的力气最大，因此对德行崇高的人尊称为“龙象”，如“西来龙象”，那是指从西方来的高僧。古印度人以为下雨是龙从大海中取水而洒下人间。中国人也接受了这种说法，历本上注明几龙取水，表示今年雨量的多寡。龙王之中，有一位叫做沙竭罗龙王，他的幼女八岁时到释迦牟尼所说法的灵鹫山前，转为男身，现成佛之相。她成佛之时，为天龙八部所见。

**Comment by picklecai** [comment]
这位王仙跋吉罗陀，
这光辉的人走在前面，
背后紧紧跟着恒河。 （1.42.21）

所有的神仙连同仙人，
还有底提耶、檀那婆和罗刹，
乾闼婆、夜叉和钵罗婆罗，
连同那些紧那罗和大蛇。 （1.42.22）

乾达婆、夜叉、紧那罗、大蛇（龙）
